tests/simple/testdata/timestamps.textproto

```diff
@@ -0,0 +1,90 @@
+name: "timestamps"
+description: "Timestamp and duration tests."
+section {
+  name: "timestamp_conversions"
+  description: "Conversions of timestamps to other types."
+  test {
+    name: "toInt"
+    expr: "int(timestamp('2009-02-13T23:31:30Z'))"
+    value: { int64_value: 1234567890 }
+  }
+  test {
+    name: "toString"
+    expr: "string(timestamp('2009-02-13T23:31:30Z'))"
+    value: { string_value: "2009-02-13T23:31:30Z" }
+  }
+  test {
+    name: "toType"
+    expr: "type(timestamp('2009-02-13T23:31:30Z'))"
+    value: { type_value: "google.protobuf.Timestamp" }
+  }
+}
+section {
```

Comment:
Can you also add a copy of the tests for the exact same inputs, but with a timezone argument specified in the helper accessor?

Reply:
Done.